From 9061a24dc15d5336327a0a43373ab7b71b0a01eb Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 20 Mar 2024 17:19:54 +0100
Subject: [PATCH 0748/1695] plat-stm32mp2: conf: enable cpu_opp

Enable stm32_cpu_opp for stm32mp2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git a9bb9da0d3ddec82a7e372f24feddd77dff98630 plat-stm32mp2: conf: enable cpu_opp]
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: I09946975159ed3d3904cc59a70abc5dd22266956
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/371771
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index cd8a1d91c..b3de15376 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -124,6 +124,7 @@ endif
 
 CFG_STM32_BSEC3 ?= y
 CFG_STM32_BSEC_WRITE ?= y
+CFG_STM32_CPU_OPP ?= y
 CFG_STM32_CRYP ?= y
 CFG_STM32_EXTI ?= y
 CFG_STM32_FMC ?= y
-- 
2.47.2

